Answers.md

```diff
@@ -0,0 +1,17 @@
+1. If you saw this HTML: ```<div class="box box1 box2 box3"></div>``` which class has the most specificity weight?
+2. Describe the difference between ```display: block;``` and ```display: inline;```.
+3. While using flexbox, what axis are you using when you use the property: ```align-items: center```?
+4. What is the difference between fixed layout, adaptive layout, fluid layout, and responsive layout?
+5. Why do we need to use the CSS property max-width on the outter most container in a responsive website?
+
+1. They are all elements worth one point but I would select ```.box3```
```

Comment:
Great answers for all of the questions! Yes, box3 would be correct because if everything has the same specificity, then it goes to the last element.